Mathlib/Algebra/Order/Floor.lean

```diff
@@ -499,6 +499,21 @@ theorem floor_div_eq_div (m n : ℕ) : ⌊(m : α) / n⌋₊ = m / n := by
 
 end LinearOrderedSemifield
 
+section LinearOrderedField
+variable [LinearOrderedField α] [FloorSemiring α] {a b : α}
+
+lemma ceil_lt_mul (ha : 0 ≤ a) (hb : 1 < b) (h : (b - 1)⁻¹ ≤ a) : ⌈a⌉₊ < b * a := by
+  rw [← sub_pos] at hb
+  calc
+    ⌈a⌉₊ < a + 1 := ceil_lt_add_one ha
+    _ = a + (b - 1) * (b - 1)⁻¹ := by rw [mul_inv_cancel₀]; positivity
+    _ ≤ a + (b - 1) * a := by gcongr; positivity
+    _ = b * a := by rw [sub_one_mul, add_sub_cancel]
+
+lemma ceil_lt_two_mul (ha : 1 ≤ a) : ⌈a⌉₊ < 2 * a :=
```

Comment:
`⌈a⌉₊ <= 2 * a` for `0.5 <= a` is something I've needed a few times. Can leave it as TODO for now